pkg/liqonet/tunnel/wireguard/driver.go

```diff
@@ -351,8 +355,22 @@ func (w *Wireguard) Close() error {
 	return fmt.Errorf("failed to delete existng WireGuard device: %w", err)
 }
 
+// runWgUserCmd runs the wg command with the given arguments.
+func runWgUserCmd(cmd *exec.Cmd) {
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		if err != nil {
+			outStr, errStr := stdout.String(), stderr.String()
+			fmt.Printf("out:\n%s\nerr:\n%s\n", outStr, errStr)
```

Comment:
Can we use klog here?

Reply:
Since that output can have multiple lines is better to use `fmt`